synphot/utils.py

```diff
@@ -282,55 +290,65 @@ def download_data(cdbs_root, verbose=True, dry_run=False):
 
     """
     from .config import conf  # Avoid potential circular import
+    BASE_HOST = 'http://ssb.stsci.edu/cdbs/'
 
-    if not os.path.exists(cdbs_root):
-        os.makedirs(cdbs_root, exist_ok=True)
-        if verbose:  # pragma: no cover
-            print('Created {}'.format(cdbs_root))
-    elif not os.path.isdir(cdbs_root):
-        raise OSError('{} must be a directory'.format(cdbs_root))
+    if path_root is not None:
+        if not os.path.exists(path_root):
+            os.makedirs(path_root, exist_ok=True)
+            if verbose:  # pragma: no cover
+                print('Created {}'.format(path_root))
+        elif not os.path.isdir(path_root):
+            raise OSError('{} must be a directory'.format(path_root))
 
-    host = 'http://ssb.stsci.edu/cdbs/'
-    file_list = []
+        if not path_root.endswith(os.sep):
+            path_root += os.sep
 
-    if not cdbs_root.endswith(os.sep):
-        cdbs_root += os.sep
+    file_list = []
 
     # See https://github.com/astropy/astropy/issues/8524
     for cfgitem in conf.__class__.__dict__.values():
         if (not isinstance(cfgitem, ConfigItem) or
                 not cfgitem.name.endswith('file')):
             continue
 
-        url = cfgitem()
+        url = cfgitem.defaultvalue
```

Comment:
This line fixes #188 but also changes the behavior reflected in the diff for `test_download_data`. I think the behavior change is a small price to pay for this fix.